director/commands/celery.py

```diff
@@ -57,5 +57,6 @@ def worker(dev_mode, worker_args):
 def flower(ctx, flower_args):
     """Start the flower instance"""
     broker = ctx.app.config["CELERY_CONF"]["broker_url"]
-    args = ["flower", "-b", broker] + list(flower_args)
+    args = ["celery", "-b", broker, "flower"]
+    args += list(flower_args)
```

Comment:
IMO, I don't think this is better.
Actually your PR should focus on the title you gave: "Celery 5 upgrade and compliance".

Reply:
Yes, this is a mandatory change which is introduced by celery 5. 

I have linked the documentation for the breaking CLI changes here [Upgrading from Celery 4.x](https://docs.celeryproject.org/en/stable/history/whatsnew-5.0.html#upgrading-from-celery-4-x).
Unfortunately their project/docs site is currently down: https://github.com/celery/celery/issues/7351, It is however accessible using the archive on [wayback machine](https://web.archive.org/web/20210615214611/https://docs.celeryproject.org/en/stable/history/whatsnew-5.0.html#upgrading-from-celery-4-x).


- Reference to celery CLI changes in [flower 1.0.0 release notes](https://pypi.org/project/flower/)
- Previous discussion in a closed PR: https://github.com/ovh/celery-director/pull/115

Reply:
related [stackoverflow question](https://stackoverflow.com/questions/67145004/the-support-for-this-usage-was-removed-in-celery-5-0-instead-you-should-use-a)